api/routes/application/routes.py

```diff
@@ -121,9 +123,23 @@ def submit(self, application_id):
                 {"application": application_with_form_json},
             )
             return {
-                "id": application_with_form_json["id"],
+                "id": application_id,
                 "reference": application_with_form_json["reference"],
                 "email": account.email,
             }, 201
         except KeyError as e:
             return {"code": 404, "message": str(e)}
+        except NotificationError as e:
+            current_app.logger.exception(
+                "Notification error on sending SUBMIT notification for"
+                f" application {application_id}"
+            )
+            # return {"code": 500, "message": str(e)}
```

Comment:
think this is a hangover 👀  (also line 144)

Reply:
Thanks - both gone